### DISCUSSION TITLE
請求協助選字的lua程式

### DISCUSSION THREAD

**Discussion by MarkenChen** [rime/home#1499] [General]
我是使用無蝦米輸入法，我想要一個功能，當同一個編碼有2個以上時，能加一碼選第2個字。
例如 pi 的候選字有1.「譬」和2.「片」，所以輸入 piv 時，就改成 「片」變成第1個後選字，然後就可以上字。
當然，前提是字庫中沒有 piv 這個編碼的字，如果有，就還是以字庫中的優先，這個加1碼的功能就停用。
這樣就不需要在字庫中，對重覆編碼的字，一個一個加上 v 的編碼。
rime 有 lua 的擴充，因為不會 lua ，想請教高手協助，如果可以靠設定就達成，那就更好了。
謝謝

**Comment by byzod** [comment]
https://github.com/iDvel/rime-ice/blob/main/radical_pinyin.schema.yaml

但是我看不懂怎么调用的🐷

**Reply by MarkenChen** [reply]
謝謝

**Comment by mirtlecn** [comment]


**Reply by MarkenChen** [reply]
謝謝，會再研究一下
